test/unit/range/container/constexpr_vector_test.cpp

```diff
@@ -0,0 +1,364 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <utility>
+
+#include <seqan3/range/container/concept.hpp>
+#include <seqan3/range/container/constexpr_vector.hpp>
+#include <seqan3/std/ranges>
+
+using namespace seqan3;
+
+// standard construction.
+TEST(constexpr_vector, standard_construction)
+{
+    EXPECT_TRUE((std::is_default_constructible_v<constexpr_vector<char, 4>>));
+    // EXPECT_TRUE((std::is_trivially_default_constructible_v<constexpr_vector<char, 4>>));
```

Comment:
It's not?

Reply:
I don't know :see_no_evil:

Reply:
A trivially default constructible class is a class (defined with class, struct or union) that:
* uses the implicitly defined default constructor.
* has no virtual members.
* **has no non-static data members with brace- or equal- initializers.**
* its base class and non-static data members (if any) are themselves also trivially default constructible types.

If you remove the brace initialisation from `data` and `sz`, it will work. But of course this breaks the other tests and is probably not what we want?

Reply:
Thanks for the explanations :)